safe/definitions/provenance.py

```diff
@@ -0,0 +1,186 @@
+    'name': tr('Action Checklist'),
+    'provenance_key': 'action_checklist'
+}
+provenance_aggregation_keywords = {
+    'key': 'provenance_aggregation_keywords',
+    'name': tr('Aggregation Keywords'),
+    'provenance_key': 'aggregation_keywords'
+}
+provenance_aggregation_layer = {
+    'key': 'provenance_aggregation_layer',
+    'name': tr('Aggregation Layer'),
+    'provenance_key': 'aggregation_layer'
+}
+provenance_aggregation_layer_id = {
+    'key': 'provenance_aggregation_layer_id',
+    'name': tr('Aggregation Layer Id'),
+    'provenance_key': 'aggregation_layer_id'
+}
+provenance_analysis_extent = {
+    'key': 'provenance_analysis_extent',
+    'name': tr('Analysis Extent'),
+    'provenance_key': 'analysis_extent'
+}
+provenance_analysis_question = {
+    'key': 'provenance_analysis_question',
+    'name': tr('Analysis Question'),
+    'provenance_key': 'analysis_question'
+}
+provenance_data_store_uri = {
+    'key': 'provenance_data_store_uri',
+    'name': tr('Data Store Uri'),
+    'provenance_key': 'data_store_uri'
+}
+provenance_duration = {
+    'key': 'provenance_duration',
+    'name': tr('Duration'),
+    'provenance_key': 'duration'
+}
+provenance_end_datetime = {
+    'key': 'provenance_end_datetime',
+    'name': tr('End Datetime'),
+    'provenance_key': 'end_datetime'
+}
+provenance_exposure_keywords = {
+    'key': 'provenance_exposure_keywords',
+    'name': tr('Exposure Keywords'),
+    'provenance_key': 'exposure_keywords'
+}
+provenance_exposure_layer = {
+    'key': 'provenance_exposure_layer',
+    'name': tr('Exposure Layer'),
+    'provenance_key': 'exposure_layer'
+}
+provenance_exposure_layer_id = {
+    'key': 'provenance_exposure_layer_id',
+    'name': tr('Exposure Layer Id'),
+    'provenance_key': 'exposure_layer_id'
+}
+provenance_gdal_version = {
+    'key': 'provenance_gdal_version',
+    'name': tr('Gdal Version'),
+    'provenance_key': 'gdal_version'
+}
+provenance_hazard_keywords = {
+    'key': 'provenance_hazard_keywords',
+    'name': tr('Hazard Keywords'),
+    'provenance_key': 'hazard_keywords'
+}
+provenance_hazard_layer = {
+    'key': 'provenance_hazard_layer',
+    'name': tr('Hazard Layer'),
+    'provenance_key': 'hazard_layer'
+}
+provenance_hazard_layer_id = {
+    'key': 'provenance_hazard_layer_id',
+    'name': tr('Hazard Layer Id'),
+    'provenance_key': 'hazard_layer_id'
+}
+provenance_host_name = {
+    'key': 'provenance_host_name',
+    'name': tr('Host Name'),
+    'provenance_key': 'host_name'
+}
+provenance_impact_function_name = {
+    'key': 'provenance_impact_function_name',
+    'name': tr('Impact Function Name'),
+    'provenance_key': 'impact_function_name'
+}
+provenance_impact_function_title = {
+    'key': 'provenance_impact_function_title',
+    'name': tr('Impact Function Title'),
+    'provenance_key': 'impact_function_title'
+}
+provenance_inasafe_version = {
+    'key': 'provenance_inasafe_version',
+    'name': tr('Inasafe Version'),
```

Comment:
Oops, I am planning to fix the case in this file, since I was generating it using script. :D 

Thanks for the catch :D